### PR TITLE
CB-10522: Event binding in Hello World is misleading

### PR DESCRIPTION
**Pull request by sarangan12**
The current event binding in the Hello World example is slightly misleading. So, fixing it. 

@nikhilkh @dblotsky @riknoll @rkatyal Can you please review and merge this PR?

### CODE REVIEW

**Comment by dblotsky** [comment]
LGTM.

**Comment by sarangan12** [comment]
@axemclion  Can you review this PR?

**Comment by dblotsky** [comment]
Will this also be going out as an email to the dev@ list?

**Comment by riknoll** [comment]
@dblotsky yes. Definitely needs dev list approval.

File: www/js/index.js [hunk 1]
```diff
@@ -19,22 +19,20 @@
...
     // Application Constructor
     initialize: function() {
-        this.bindEvents();
-    },
-    // Bind Event Listeners
-    //
-    // Bind any events that are required on startup. Common events are:
-    // 'load', 'deviceready', 'offline', and 'online'.
-    bindEvents: function() {
         document.addEventListener('deviceready', this.onDeviceReady, false);
```
Comment (axemclion) [inline www/js/index.js]: Do we wanna do a this.onDeviceReady.bind(this) here ?

Comment (dblotsky) [inline www/js/index.js]: Any variables in `app`'s scope are already in `onDeviceReady`'s scope.

**Comment by sarangan12** [comment]
@axemclion Can you check now?

**Comment by devgeeks** [comment]
LGTM. That third layer of binding always seemed confusing.

:+1:

**Comment by dblotsky** [comment]
Cool, seems like we've reached a consensus. Merging!

**Pull request merged**